librz/core/cmd/cmd_help.c

```diff
@@ -434,63 +435,8 @@ RZ_API void rz_core_cmd_help_calc_expr(RZ_NONNULL RzCore *core, RZ_NONNULL const
 }
 
 RZ_IPI int rz_cmd_help(void *data, const char *input) {
-	/*
-		RzCore *core = (RzCore *)data;
-		RzIOMap *map;
-		const char *k;
-		RzListIter *iter;
-		char *p, out[128] = RZ_EMPTY;
-		ut64 n;
-		int i;
-		RzList *tmp;
-
-		switch (input[0]) {
-		case 'b': // "?b"
-			} else if (input[1] == 't' && input[2] == 'w') { // "?btw"
-				if (rz_num_between(core->num, input + 3) == -1) {
-					RZ_LOG_ERROR("core: Usage: ?btw num|(expr) num|(expr) num|(expr)\n");
-				}
-			}
-		case '@': // "?@"
-			if (input[1] == '@') {
-				if (input[2] == '@') {
-					rz_core_cmd_help(core, help_msg_at_at_at);
-				} else {
-					rz_core_cmd_help(core, help_msg_at_at);
-				}
-			} else {
-				rz_core_cmd_help(core, help_msg_at);
-			}
-		case '?': // "??"
-			if (input[1] == '?') {
-				if (input[2] == '?') { // "???"
-					rz_core_clippy_print(core, "What are you doing?");
-					return 0;
-				}
-				if (input[2]) {
-					if (core->num->value) {
-						rz_core_cmd(core, input + 1, 0);
-					}
-					break;
-				}
-				rz_core_cmd_help(core, help_msg_question);
-				return 0;
-			} else if (input[1]) {
-				if (core->num->value) {
-					core->num->value = rz_core_cmd(core, input + 1, 0);
-				}
-			} else {
-				if (core->num->dbz) {
-					RZ_LOG_ERROR("core: RzNum ERROR: Division by Zero\n");
-				}
-				rz_cons_printf("%" PFMT64d "\n", core->num->value);
-			}
-			break;
-		case '\0': // "?"
-		default:
-			break;
-		}
-		*/
+	// TODO: I don't know how to remove this function.
+	// It's called in cmd.c file in rz_core_cmd_init
```

Comment:
open an issue for this

Reply:
I tried removing it this time and it worked. Maybe it failed before because not all commands were ported.